config.go

```diff
@@ -479,6 +481,24 @@ func loadConfig() (*config, []string, error) {
 		return nil, nil, err
 	}
 
+	// Set the default policy for relaying non-standard transactions
+	// according to the default of the active network. The set
+	// configuration value takes precedence over the default value for the
+	// selected network.
+	switch {
+	case cfg.RelayNonStd && cfg.RejectNonStd:
+		str := "%s: rejectnonstd and relaynonstd cannot be used " +
+			"together -- choose only one"
+		err := fmt.Errorf(str, funcName)
+		fmt.Fprintln(os.Stderr, err)
+		fmt.Fprintln(os.Stderr, usageMessage)
+		return nil, nil, err
+	case cfg.RejectNonStd && activeNetParams.RelayNonStdTxs:
```

Comment:
The logic in this section is seems a little hard to follow in my opinion.  How about the following which I think is a little clearer.  Start with the value configured by the network params, then override it if one of the flags is set.

```diff
diff --git a/config.go b/config.go
index 1b04b2f..b3af9c5 100644
--- a/config.go
+++ b/config.go
@@ -485,6 +485,7 @@ func loadConfig() (*config, []string, error) {
        // according to the default of the active network. The set
        // configuration value takes precedence over the default value for the
        // selected network.
+       relayNonStd := activeNetParams.RelayNonStdTxs
        switch {
        case cfg.RelayNonStd && cfg.RejectNonStd:
                str := "%s: rejectnonstd and relaynonstd cannot be used " +
@@ -493,11 +494,12 @@ func loadConfig() (*config, []string, error) {
                fmt.Fprintln(os.Stderr, err)
                fmt.Fprintln(os.Stderr, usageMessage)
                return nil, nil, err
-       case cfg.RejectNonStd && activeNetParams.RelayNonStdTxs:
-               cfg.RelayNonStd = false
-       case cfg.RelayNonStd && !activeNetParams.RelayNonStdTxs:
-               cfg.RelayNonStd = true
+       case cfg.RejectNonStd:
+               relayNonStd = false
+       case cfg.RelayNonStd:
+               relayNonStd = true
        }
+       cfg.RelayNonStd = relayNonStd

        // Append the network type to the data directory so it is "namespaced"
        // per network.  In addition to the block database, there are other
```